contrib/qbe/patches/fix_asm_keyword.patch

```diff
@@ -0,0 +1,62 @@
+--- a/arm64/emit.c	2024-01-29 10:24:41.000000000 +0100
```

Comment:
add a link to where it's from

Reply:
enjoy sending it :3

Reply:
sure, though i would just send a new patch anyway because they are discussing random stuff in there